docs/docs/getting-started/installation/minikube.md

```diff
@@ -0,0 +1,84 @@
+---
+title: Minikube
+sidebar_label: Minikube
+---
+
+import useBaseUrl from '@docusaurus/useBaseUrl';
+
+The goal of this document is to provide an overview of how to run Airy Core on
+your local machine using [minikube](https://minikube.sigs.k8s.io/).
+
+## Creating a cluster
+
+First download and install minikube from their [release page](https://github.com/kubernetes/minikube/releases) and the [Airy CLI](cli/installation.md).
+Now you can run this command, which will create a new minikube cluster on your system and install Airy core on it:
+
+```bash
+airy create --provider=minikube
+```
+
+This will print URLs for accessing the UIs and APIs as seen in this recording:
+
+<div id="asciicast-A9sTPeYDKJ2ky2hauwumzFEwQ" ></div>
+
+import Script from "@site/src/components/Script";
+
+<Script data-rows="12" id="asciicast-A9sTPeYDKJ2ky2hauwumzFEwQ" src="https://asciinema.org/a/A9sTPeYDKJ2ky2hauwumzFEwQ.js"></Script>
```

Comment:
Component to add embedded asciinema videos 🎉

Reply:
very cool! I'd call the component what it does though. `<Asciinema/>`

Reply:
ah now I see why you called it script though.... maybe we should specialise though (no strong opinion)

Reply:
Generally I would agree but this component can power any script. I think the description I left above is a bit misleading.